lib/internal/modules/esm/hooks.js

```diff
@@ -535,6 +550,23 @@ class HooksProxy {
     }
   }
 
+  #postMessageToWorker(method, type, transferList, ...args) {
+    this.waitForWorker();
+    MessageChannel ??= require('internal/worker/io').MessageChannel;
+    const { port1: fromHooksThread, port2: toHooksThread } = new MessageChannel();
+
+    // Pass work to the worker.
+    debug(`post ${type} message to worker`, { method, args, transferList });
+    const usedTransferList = [toHooksThread];
+    if (transferList) {
+      ArrayPrototypePushApply(usedTransferList, transferList);
+    }
+    this.#portToHooksThread.postMessage(
+      { __proto__: null, method, args, lock: this.#lock, port: toHooksThread }, usedTransferList);
```

Comment:
nit (whitespace & sorting)

```suggestion
    }

    this.#portToHooksThread.postMessage(
      {
        __proto__: null,
        args,
        lock: this.#lock,
        method,
        port: toHooksThread,
      },
      usedTransferList,
    );
```